### PR TITLE
Upgrade to Kotlin 1.7.10

### PR DESCRIPTION
**Pull request by C2H6O**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,4 @@
-VERSION=0.8.2
+VERSION=0.8.3
 
-kotlin.mpp.enableGranularSourceSetsMetadata=true
```
Comment (C2H6O) [inline gradle.properties:3]: Gave me a warning that this is enabled by default and can be safely removed

**Review by DanielRBaird — APPROVED** [review]
Awesome, thanks for putting this up.

**Review by benasher44 — APPROVED** [review]

**Comment by C2H6O** [comment]
I think one of y'all will have to merge this in because I haz no access

**Pull request merged**